meson_options.txt

```diff
@@ -0,0 +1,3 @@
+option('usewebcam', type: 'boolean', value: false, description: 'Build with webcam support')
+option('usemididrum', type: 'boolean', value: false, description: 'Build with midi drum support')
+option('usewebserver', type: 'boolean', value: false, description: 'Build with web server support')
```

Comment:
There are some more options available in cmake (libxml2.6 vs libxml3.0 for example)

Reply:
In the meson build, the choice between libxml2.6 vs 3.0 is automated, based on the version installed.
Preference is given to v3.0, with 2.6 as fallback.
See line 51 in meson.build